src/adapters/lokijs/worker/encodeQuery/index.js

```diff
@@ -86,6 +87,19 @@ const noNullComparisons: OperatorFunction => OperatorFunction = operator => valu
   $and: [operator(value), weakNotEqual(null)],
 })
 
+const like: OperatorFunction = value => {
+  if (typeof value === 'string') {
+    const regexp = `^${value}$`.replace(/%/g, '.*').replace(/_/g, '.')
+    return {
+      $regex: new RegExp(regexp, 'i'),
+    }
+  }
+
+  return {
+    $regex: new RegExp('(?!)', 'i'),
+  }
```

Comment:
hm, does this even make sense? I thought only string values are permitted as LIKE operand